.travis.yml

```diff
@@ -6,12 +6,10 @@ cache: bundler
 language: ruby
 rvm:
 - ruby-head
-- 2.2
-- 2.1
-- 2.0.0
```

Comment:
The problem lies in the development dependencies. We could remove at least `metric_fu` which depends on `activesupport` and `rails_best_practices` which have pretty strict ruby requirements themselves. `metric_fu` doesn't seem to be used in the CI as far as I have gone through the code (but I'm not too familiar with it yet).
I'd in general favor a lighter list of dev dependencies anyway, IMO also `simplecov`, `simplecov-rcov-text`, `guard`, `guard-minitest`, `ruby-prof` and `metric_fu` could be removed without harm.